source/usage-examples/count.txt

```diff
@@ -3,3 +3,47 @@ Count Documents
 ===============
 
 .. default-domain:: mongodb
+
+You can count the number of documents in a collection using the
+``EstimatedDocumentCount()`` and ``CountDocuments()`` methods. 
+
+The following example uses the ``EstimatedDocumentCount()`` method to
+count the number of documents in the ``movies`` collection. Then, the
+example used the ``CountDocuments()`` method to count the number of
+documents in the ``movies`` collection where the ``countries`` field is
+"China".
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/count.go
+   :start-after: begin countDocuments
+   :end-before: end countDocuments
+   :emphasize-lines: 4-5
+   :language: go
+   :dedent:
+
+Click here <TODO> to see a fully runnable example.
```

Comment:
See [here](https://github.com/mongodb/docs-golang/blob/master/source/usage-examples/insertOne.txt#L28) for an example on replacing this TODO. It is known these links won't work until the PR is merged.